ehitchhiking-ui/angular.json

```diff
@@ -28,7 +28,8 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.sass"
+              "src/assets/css/fonts.css",
```

Comment:
pls add "src/assets/css/fonts.css" to the "test" too

Add "src/styles.sass" and  remove "src/app/shared/common/styles/colour-theme.sass" here. You should import src/app/shared/common/styles/colour-theme.sass in the styles.sass

Reply:
done